fcli-other/fcli-functional-test/src/ftest/groovy/com/fortify/cli/ftest/ssc/SSCUserSpec.groovy

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright 2023 Open Text.
+ *
+ * The only warranties for products and services of Open Text 
+ * and its affiliates and licensors ("Open Text") are as may 
+ * be set forth in the express warranty statements accompanying 
+ * such products and services. Nothing herein should be construed 
+ * as constituting an additional warranty. Open Text shall not be 
+ * liable for technical or editorial errors or omissions contained 
+ * herein. The information contained herein is subject to change 
+ * without notice.
+ */
+package com.fortify.cli.ftest.ssc
+
+import static com.fortify.cli.ftest._common.spec.FcliSessionType.SSC
+
+import com.fortify.cli.ftest._common.Fcli
+import com.fortify.cli.ftest._common.spec.FcliBaseSpec
+import com.fortify.cli.ftest._common.spec.FcliSession
+import com.fortify.cli.ftest._common.spec.Prefix
+import com.fortify.cli.ftest.ssc._common.SSCAppVersion
+
+import spock.lang.AutoCleanup
+import spock.lang.Requires
+import spock.lang.Shared
+
+@Prefix("ssc.user") @FcliSession(SSC) 
+class SSCUserSpec extends FcliBaseSpec {
+    
+    def "list"() {
+        def args = "ssc user list"
+        when:
+            def result = Fcli.run(args)
+        then:
+            verifyAll(result.stdout) {
+                size()>0
+                it[0].replace(' ', '').equals("IdEntitynameDisplaynameTypeEmailIsldap")
+                it.any { it.startsWith(" 1") }
+            }
+    }
+    
+    def "get.byId"() {
+        def args = "ssc user get 1"
+        when:
+            def result = Fcli.run(args)
+        then:
+            verifyAll(result.stdout) {
+                size()>0
+                it[1].equals("id: 1")
+            }
+    }
+    
+    def "get.byName"() {
+        def args = "ssc user get admin"
+        when:
+            def result = Fcli.run(args)
+        then:
+            verifyAll(result.stdout) {
+                size()>0
+                it[1].equals("id: 1")
+            }
+    }
+    
+    def "get.byMail"() {
+        def args = "ssc user get my_email@fortify.com"
```

Comment:
This is only true for FH UAT environment; maybe better to again use `@StepWise` and take the email address from the output of the `get` command above.